newrelic-agent/src/main/java/com/newrelic/agent/AgentCommandLineParser.java

```diff
@@ -45,7 +45,6 @@
 
         commandDescriptions = new HashMap<>();
         commandDescriptions.put(DEPLOYMENT_COMMAND, "[OPTIONS] [description]  Records a deployment");
-        commandDescriptions.put(INSTRUMENT_COMMAND, "[OPTIONS]                Validates a custom instrumentation xml configuration file.");
```

Comment:
Great catch! It looks like this was removed in Saxon's original prototype and I have no idea why. I'm going to add it back since as far as I know we shouldn't be changing anything related to the instrumentation validator.